src/frontend/src/pages/Calendar/CalendarPage.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+import { useState } from 'react';
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import PageLayout from '../../components/PageLayout';
+import { DesignReview } from 'shared';
+import MonthSelector from './CalendarComponents/MonthSelector';
+import DayCard from './CalendarComponents/CalendarDayCard';
+import FillerCard from './CalendarComponents/FillerCalendarDayCard';
+import { DAY_NAMES, EnumToArray, testDesignReview1 } from '../../utils/design-review.utils';
+import ActionsMenu from '../../components/ActionsMenu';
+
+const CalendarPage = () => {
+  const theme = useTheme();
+
+  const [displayMonthYear, setDisplayMonthYear] = useState<Date>(new Date());
+
+  const EventDict = new Map<Number, DesignReview[]>();
+  // TODO remove during wire up ticket
+  EventDict.set(new Date().getDate(), [testDesignReview1]);
+
+  const startOfEachWeek = [0, 7, 14, 21, 28, 35];
+
+  const isDayInDifferentMonth = (day: number, week: number) => {
+    return day < week - 7 || day < 1 || day > week + 7;
+  };
+
+  const daysInMonth = (month: Date) => {
+    return new Date(month.getFullYear(), month.getMonth() + 1, 0).getDate();
+  };
+
+  const paddingDays = (month: Date) => {
+    return new Date(month.getFullYear(), month.getMonth(), 0).getDay();
+  };
+
+  const paddingArrayStart = [...Array<number>(paddingDays(displayMonthYear)).keys()].map(
+    (day) =>
+      daysInMonth(new Date(displayMonthYear.getDate(), displayMonthYear.getMonth() - 1, displayMonthYear.getFullYear())) -
+      day
+  );
+  const paddingArrayEnd = [
+    ...Array<number>(7 - ((daysInMonth(displayMonthYear) + paddingDays(displayMonthYear)) % 7)).keys()
+  ].map((day) => day + 1);
+  const daysThisMonth = paddingArrayStart
+    .concat([...Array(daysInMonth(displayMonthYear)).keys()])
+    .map((day) => day + 1)
+    .concat(paddingArrayEnd.length < 7 ? paddingArrayEnd : []);
+
+  const unconfirmedDRSDropdown = (
+    <ActionsMenu
+      title="My Unconfirmed DRS"
+      buttons={[
+        {
+          title: 'Mock Review #1',
+          onClick: () => {},
+          disabled: false
+        },
+        {
+          title: 'Mock Review #2',
+          onClick: () => {},
+          disabled: false
+        }
+      ]}
+    >
+      My Unconfirmed DRs
+    </ActionsMenu>
+  );
+
+  return (
+    <PageLayout title="Design Review Calendar" headerRight={unconfirmedDRSDropdown}>
+      <MonthSelector displayMonth={displayMonthYear} setDisplayMonth={setDisplayMonthYear} />
```

Comment:
Can you see how moving the Month Selector to be to the left of the unconfirmedDRs dropdown looks, we're wasting a lot of space at the top of the page rn
<img width="942" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/6cb13849-a7b4-49b8-a4a3-fc6d16f322fb">